snuba_sdk/entity.py

```diff
@@ -11,11 +12,12 @@ class InvalidEntityError(Exception):
     pass
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True, repr=False)
 class Entity(Expression):
     name: str
     alias: Optional[str] = None
     sample: Optional[float] = None
+    data_model: Optional[EntityModel] = field(hash=False, default=None)
```

Comment:
I had to add the `hash=False` here. EntityModels have ColumnSets which have sets which are unhashable. If I left this, Entities would be unhashable, and then Columns would also be unhashable since they sometimes contain Entities.